pkg/utils/output.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"sigs.k8s.io/yaml"
+)
+
+type Output interface {
+	EnvFormat() string
+}
+
+func RenderOutput[T Output](outputFormat string, data T) (string, error) {
```

Comment:
It's not a big issue so if you are worried to break things I guess we can keep it in.
I just feel the generics make the signature more complex than it needs to be for no real generic-benefit as only a single branch even uses it.

Reply:
I could look into it. I personally don't love having "util" style packages depend on other packages (in this case, `utils` would need to depend on `awsUtil` but I'll see if it works out. It would be nice to use `awsUtil.AWSCredentialsResponse` instead of `T` here.